force-app/main/default/classes/AccountCallouts.cls

```diff
@@ -0,0 +1,26 @@
+public with sharing class AccountCallouts {
+    
+    public static Map<Integer, String> executeHttpRequest(String endpoint, String method,String bodyData, Boolean hasBody) {
+        
+        Map<Integer, String> responseMap = new Map<Integer, String>();
+        HttpRequest request = new HttpRequest();
+        request.setEndpoint('callout:ReturnContact' + endpoint);
+        request.setHeader('content-type', 'application/json; charset=UTF-8');
+        request.setHeader('Accept', 'application/json'); 
+        if(hasBody){
```

Comment:
you can check for bodyData to determine if there is some body. You don't need to have additional parameter for the method